src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -148,6 +148,14 @@ function OptionRowLHN(props) {
 
                             props.onSelectRow(optionItem, popoverAnchor);
                         }}
+                        // Prevent losing Composer focus
```

Comment:
Let's take this comment inside and above `preventDefault()`

Reply:
Let's make sure to test it does not add regression if focus is on other element then composer. cc @Santhosh-Sellavel